Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestRunIndexViewController.swift

```diff
@@ -20,6 +20,8 @@ enum Status {
 class TestRunIndexViewController: UIViewController, UITextViewDelegate {
    
     @IBOutlet weak var cancelRun: UIBarButtonItem!
+    @IBOutlet weak var blockedButton: UIButton!
```

Comment:
If you remove these make sure that the connections set up in the storyboard connections inspector are also cleared.